apps/earn-protocol/app/portfolio/[walletAddress]/page.tsx

```
@@ -72,7 +82,15 @@ const PortfolioPage = async ({ params }: PortfolioPageProps) => {
       portfolioPositionsHandler({ position, vaultsList: vaults, config, walletAddress }),
     ),
   )
-  const vaultsDecorated = decorateCustomVaultFields({ vaults, systemConfig: config })
+  const vaultsDecorated = decorateCustomVaultFields({
+    vaults,
+    systemConfig: config,
+    decorators: {
+      arkInterestRatesMap: interestRatesResults.reduce((acc, curr) => {
+        return { ...acc, ...curr }
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)